addons/l10n_at/data/account_tax_report_data.xml

```diff
@@ -13,6 +13,49 @@
             </record>
         </field>
         <field name="line_ids">
+            <record id="tax_report_line_l10n_at_non_tva_sale_report_title" model="account.report.line">
+                <field name="name">3. Aufschlüsselung für die Zusammenfassende Meldung (ZM)</field>
+                <field name="sequence" eval="5"/>
+                <field name="aggregation_formula">(AT_ZM_IGL.balance + AT_ZM_IGL3.balance + AT_ZM_DL.balance)</field>
+                <field name="children_ids">
+                    <record id="tax_report_line_l10n_at_tva_line_3_zm_igl" model="account.report.line">
+                        <field name="name">Innergemeinschaftliche Lieferungen</field>
+                        <field name="code">AT_ZM_IGL</field>
+                        <field name="expression_ids">
+                            <record id="tax_report_line_l10n_at_tva_line_3_zm_igl_tag" model="account.report.expression">
+                                <field name="label">balance</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">KZ ZM IGL</field>
```

Comment:
Hmm, seems to be related to the tag name which vanished, I will need to take a close look to this section again, on the first sight, it would be more something like `AT_ZM_IGL`